examples/bsc-sdk/src/node/evm/patch/mod.rs

```diff
@@ -0,0 +1,717 @@
+use alloy_consensus::Transaction;
+use alloy_primitives::{address, b256, Address, B256, U256};
+use lazy_static::lazy_static;
+use reth_evm::block::BlockExecutionError;
+use reth_revm::{db::states::StorageSlot, State};
+use revm::Database;
+use std::{collections::HashMap, str::FromStr};
+use tracing::trace;
+
+struct StoragePatch {
+    address: Address,
+    storage: HashMap<U256, U256>,
+}
+
+lazy_static! {
```

Comment:
let's convert this into a Lazy type and not use the lazy_static crate for this, see chainspec statics for example